src/code/fault.c

```diff
@@ -48,9 +48,9 @@
 // just above (the exact upper bound depends on the block numbers assigned to
 // extern variables declared in headers).
 #if OOT_DEBUG
-#pragma increment_block_number 20
+#pragma increment_block_number 0
 #else
-#pragma increment_block_number 25
+#pragma increment_block_number 20
 #endif
```

Comment:
optional: not sure how this `#if OOT_DEBUG` got here (probably my mistake) but `increment_block_number` has no effect for debug. Could you clean this up?

Reply:
I believe it to be correct; fault.o is compiled with -g3